bower.json

```diff
@@ -7,6 +7,7 @@
     "o-colors": "^3.5.4",
```

Comment:
you can remove all these o-colors, o-icons etc dependencies too. The fewer components explictly declare dependency on them, the easier it is to roll out new versions